bgpd/bgp_bfd.c

```diff
@@ -329,19 +329,22 @@ static int bgp_bfd_dest_update(ZAPI_CALLBACK_ARGS)
 				&remote_cbit, vrf_id);
 
 	if (BGP_DEBUG(zebra, ZEBRA)) {
+		struct vrf *vrf;
 		char buf[2][PREFIX2STR_BUFFER];
+
+		vrf = vrf_lookup_by_id(vrf_id);
 		prefix2str(&dp, buf[0], sizeof(buf[0]));
 		if (ifp) {
 			zlog_debug(
-				"Zebra: vrf %u interface %s bfd destination %s %s %s",
-				vrf_id, ifp->name, buf[0],
-				bfd_get_status_str(status),
+				"Zebra: vrf %s(%u) interface %s bfd destination %s %s %s",
+				vrf ? vrf->name : "?", vrf_id, ifp->name,
```

Comment:
I see we use `Unknown` instead of `?` for such cases.

Reply:
fixed and extended to other places as well.